lib/schemas/global/search/apiDefendantSummary.json

```diff
@@ -1,88 +1,92 @@
-      "$ref": "http://justice.gov.uk/core/courts/external/apiCourtsDefinitions.json#/definitions/nino"
-    },
-    "defendantASN": {
-      "description": "The police arrest summons number when the defendant is a person",
-      "type": "string"
-    },
-    "defendantName": {
-      "description": "The name of the defendant",
-      "type": "string"
-    },
-    "defendantFirstName": {
-      "description": "The firstname of the defendant",
-      "type": "string"
-    },
-    "defendantMiddleName": {
-      "description": "The middle name of the defendant",
-      "type": "string"
-    },
-    "defendantLastName": {
-      "description": "The last name of the defendant",
-      "type": "string"
-    },
-    "defendantDOB": {
-      "description": "The person date of birth when the defendant is a person",
-      "$ref": "http://justice.gov.uk/core/courts/external/apiCourtsDefinitions.json#/definitions/datePattern"
-    },
-    "dateOfNextHearing": {
-      "description": "The date of the next hearing for the defendant",
-      "$ref": "http://justice.gov.uk/core/courts/external/apiCourtsDefinitions.json#/definitions/datePattern"
-    },
-    "proceedingsConcluded": {
-      "description": "Indicates that proceedings for the defendant have been conckuded i.e. all offences have been disposed with final results or have been withdrawn by the prosecuting authority",
-      "type": "boolean"
-    },
-    "representationOrder": {
-      "description": "The representation order for this defendant",
-      "type": "object",
-      "$ref": "http://justice.gov.uk/core/courts/external/apiRepresentationOrder.json"
-    },
-    "offenceSummary": {
-      "description": "The offences for this defendant",
-      "type": "array",
-      "minItems": 1,
-      "items": {
-        "$ref": "http://justice.gov.uk/core/courts/search/external/apiOffenceSummary.json"
-      }
-    },
-    "application_summaries": {
-      "description": "The court application for this defendant",
-      "type": "array",
-      "minItems": 1,
-      "items": {
-        "$ref": "http://justice.gov.uk/core/courts/search/external/apiApplicationSummary.json"
-      }
-    }    
-  },
-  "oneOf": [
-    {
-      "required": [
-        "defendantId",
-        "defendantName",
-        "offenceSummary"
-      ]
-    },
-    {
-      "required": [
-        "defendantId",
-        "defendantFirstName",
-        "defendantMiddleName",
-        "defendantLastName",
-        "offenceSummary"
-      ]
-    }
-  ],
-  "additionalProperties": false
-}
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "http://justice.gov.uk/core/courts/search/external/apiDefendantSummary.json",
+  "description": "A summary of defendant details",
+  "type": "object",
+  "properties": {
+    "defendantId": {
+      "description": "The identifier of the defendant",
+      "$ref": "http://justice.gov.uk/core/courts/external/apiCourtsDefinitions.json#/definitions/uuid"
+    },
+    "masterDefendantId": {
```

Comment:
This file was formatted really wonkily, something to do with character encoding maybe? All I've done is add this masterDefendantId bit.